launch_scripts/launch.sh

```
@@ -1,22 +1,22 @@
-#!/bin/bash
-
-# Setup
-echo "Setting up..."
-cd /workspaces/REAPER/src/
-source install/setup.bash
-
-# Running
-echo "Launching nodes..."
-ros2 launch controls launch.yaml &
-ros2 launch teleop launch.yaml &
-ros2 run joy joy_node
-
-# Teardown
-echo "Shutting down..."
-pkill ros2
-pkill dig
-pkill dump
-pkill distrib
-pkill drivetrain
-pkill uart
-echo "Killed background processes"
+#!/bin/bash
+
+# Setup
+echo "Setting up..."
+cd /workspaces/REAPER/src/
```

is this gonna work on the NUC? because there is no `/workspaces/` since we are not doing in the container. :(

---

I've been running everything on the nuc and it didn't complain. I suspect that the cd just fails silently